### PR TITLE
Feat/remove yarn

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: src/modules/Analytics/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ For now, we support matomo through the library [@socialgouv/matomo-next](https:/
 copy the whole module to your nextjs app and launch
 
 ```
-  yarn add @socialgouv/matomo-next
+  npm run add @socialgouv/matomo-next
```
Comment (martinratinaud) [inline src/modules/Analytics/README.md:10]: not `npm run add` but `npm install`

File: Dockerfile [hunk 2]
```diff
@@ -52,12 +52,12 @@ COPY ./docker/$ENV_FILE /usr/src/app/.env.production
 ENV NODE_ENV=production
 ENV NODE_OPTIONS='--max_old_space_size=8192'
 
-RUN yarn build
+RUN npm run build
 
 RUN rm -Rf node_modules
-RUN yarn --production
+RUN npm --production
```
Comment (martinratinaud) [inline Dockerfile:58]: `npm install --production`

File: src/modules/I18n/README.md [hunk 3]
```diff
@@ -7,8 +7,8 @@
 and run
 
 ```
-    yarn add next-i18nnext
-    yarn add -D i18next-scanner i18next-scanner-typescript
+    npm run add next-i18nnext
```
Comment (martinratinaud) [inline src/modules/I18n/README.md:10]: not npm run add but npm install

**Review by martinratinaud — APPROVED** [review]
In case it's merged after https://github.com/ambanum/OpenTermsArchive.org/pull/66

You will need to also update the new workflow file called `validate-typescript.yml`

**Comment by clementbiron** [comment]
> In case it's merged after #66
> 
> You will need to also update the new workflow file called `validate-typescript.yml`

ok well noted 👍

**Pull request merged**